### DISCUSSION TITLE
Bagaimana cara terukur, dalam mengidentifikasi sejauh mana perangkat kita mampu memproses data?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#376] [Q&A] [answered]
Bagaimana cara terukur, dalam mengidentifikasi sejauh mana perangkat kita mampu memproses data? 
Misal ada 2 juta baris dengan 100 kolom, nah karena kita nggak tahu cara mengukur keterbatasan perangkat, dijalankanlah itu model-model, tapi jadi buang waktu berjam-jam ditunggu gak selesai, ntah beneran itu model lagi mencerna atau ternyata ada salah code sehingga lama (kadang code diperbaiki tiba-tiba cepet). 

Setidaknya kalau kita bisa mengukur kemampuan perangkat komputer kita, saat berhadapan dengan data besar, kan kita bisa menggunakan sampling datanya saja, kan? Nah ini juga mau tanya, gimana menentukan jumlah sampling data yang dapat cukup dianggap mewakili populasi data asli?

Terima kasih,

**Comment by jafarhf1** [comment, marked as answer]
Dear @Hertzspr,
Terkait pertanyaan mengenai jumlah sampel bapak bisa merujuk ke link berikut: https://github.com/teamalgoritma/community/discussions/361.

Demikian yang dapat kami sampaikan. Jika jawaban ini membantu silahkan mark as answer.
